hack/util-set-github-org

```diff
@@ -3,4 +3,4 @@
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
 
 echo "Changing AppStudio Gitlab Org to \"$1\""
-yq e -i "(.configMapGenerator[].literals[] | select(. == \"*GITHUB*\")) = \"GITHUB_ORG=$1\"" $ROOT/components/has/base/kustomization.yaml
+yq e -i "(.configMapGenerator[].literals[] | select(. == \"*GITHUB*\")) = \"GITHUB_ORG=$1\"" $ROOT/components/has/overlays/development/kustomization.yaml
```

Comment:
Since we dont have a base Kustomization and we consider development overlays as default like here, then these two lines also need to be updated https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/preview.sh#L171-L172

Reply:
good catch! I'll try to update it in the next rev if I keep the current approach.  I'm also thinking about [John's suggestion](https://github.com/johnmcollier/infra-deployments/tree/separate_envs/components/has) to keep the base overlay for dev and stage.  I think it's a bit cleaner.